docs/how-to-guides/web-portal/web-portal-24-04-or-later/use-security-profiles.md

```diff
@@ -0,0 +1,74 @@
+(how-to-web-portal-use-security-profiles)=
+# How to use security profiles
+
+You can use security profiles to automatically audit and fix your managed instances on a schedule using the [Ubuntu Security Guide (USG)](https://documentation.ubuntu.com/security/docs/compliance/usg/).
+
+```{note}
+This feature is only available in self-hosted **Landscape 25.04** and later.
+```
+
+## Create a security profile
+
+On each instance you want to manage, follow the steps in [Install Ubuntu Security Guide](https://documentation.ubuntu.com/security/docs/compliance/usg/install-usg/), then:
+
+1. Edit the client configuration file `/etc/landscape/client.conf`, add this line:
+
+    ```
+    include_manager_plugins = UsgManager
+    ```
+
+    If your `client.conf` already includes an `include_manager_plugins` line, then add `UsgManager` to it. For example:
+
+    ```
+    include_manager_plugins = ScriptExecution,UsgManager
+    ```
+
+2. Restart Landscape Client:
+
+    ```
+    sudo systemctl restart landscape-client
+    ```
+
+From the web portal:
+
+1. Click **Profiles**
+2. Click **Security Profiles**
+3. Click **Add security profile**
+
+In the security profile creation form, complete the following fields:
+
+- **Profile name**: name of the profile
+- **Access group**: the access group the profile will apply to
+- **Base profile**: the security benchmark the profile will use
+- **Mode**: the profile's mode – which actions it will perform on instances
+- **Upload tailoring file**: a [benchmark customization file](https://documentation.ubuntu.com/security/docs/compliance/usg/cis-customize/). If provided, it supersedes the **Base profile**.
+- **Schedule**:
+  - **On a date**: a specific date and time at which the profile will execute once
+  - **Recurring**: a start date and end date between which the profile will execute repeatedly, with the provided number of days between executions. The recurrence cannot be more frequent than once every seven days.
+- **Association**:
+  - **Associate to all instances**: The profile will affect all instances in the same access group as the profile
+  - **Tag(s)**: Only instances having the specific tag(s), in the same access group as the profile will be affected
+  
+After you've created your security profile, you can view, edit, run, duplicate, or archive it from the **Security profiles** tag, under **Actions**.
```

Comment:
Our test account went down while I was reviewing this :sweat_smile: (so this feedback is assuming I remember correctly)

But instead did you mean that you can do those actions using the dot menu under **Actions**? Also, make sure they're in order of the options listed from dropdown (I think I saw they weren't, but not 100% sure on that one)

Reply:
I missed a couple - I've updated the list.